src/actions/cities.actions.js

```diff
@@ -0,0 +1,24 @@
+import {citiesService} from '../services';
+import {citiesConstants} from '../constants';
+
+
+export const citiesActions = {
+    getAllCities,
+    deleteAllCities
+};
+
+
+function getAllCities(){
+    const allCities = citiesService.getCities();
+    return dispatch=>{
+        if(allCities.length){
```

Comment:
`if` is redundant. It will work the same without if.

Reply:
+